qcodes/instrument_drivers/tektronix/Keithley_2600_channels.py

```diff
@@ -190,6 +191,65 @@ def get_raw(self) -> np.ndarray:  # pylint: disable=E0202
         return np.linspace(0, dt*npts, npts, endpoint=False)
 
 
+class _ParameterWithStatus(Parameter):
+    def __init__(self, *args: Any, **kwargs: Any):
+        super().__init__(*args, **kwargs)
+
+        self._measurement_status: Optional[str] = None
+
+    @property
+    def measurement_status(self) -> Optional[str]:
+        return self._measurement_status
```

Comment:
Rather than letting the measurement status be a string that one has to match against it would perhaps be better with a string enum?